src/main/java/PageObjects/BookCart_HomePage.java

```diff
@@ -14,6 +14,7 @@ public BookCart_HomePage() {
 
 	public static By bookCartLogo = By.xpath("//span[@class='mat-button-wrapper']");
 	public static By loginButton = By.xpath("//button[@class='mat-focus-indicator mat-button mat-button-base ng-star-inserted']");
+	public static By MatIconExistingUsername = By.xpath("(//span[@class='mat-button-wrapper'])[4]");
```

Comment:
use proper naming conventions.
Variable name should be start using a small letter case.